evm_loader/program/src/executor/state.rs

```diff
@@ -29,7 +29,7 @@ pub type TouchedAccounts = TreeMap<Pubkey, u64>;
 /// Persistent part of `ExecutorState`.
 #[repr(C)]
 pub struct ExecutorStateData {
-    cache: RefCell<Cache>,
+    pub cache: Cache,
```

Comment:
Please keep it as `RefCell`.
It is required for https://github.com/neonlabsorg/neon-evm/pull/553